package.json

```diff
@@ -26,7 +26,7 @@
     "@webriq-pagebuilder/sanity-plugin-schema-commerce": "^3.1.0",
     "@webriq-pagebuilder/sanity-plugin-schema-default": "^3.1.0",
     "@webriq-pagebuilder/sanity-plugin-webriq-blog": "^3.1.0",
-    "@webriq-pagebuilder/sanity-plugin-webriq-components": "^3.1.0",
+    "@webriq-pagebuilder/sanity-plugin-webriq-components": "link:.yalc/@webriq-pagebuilder/sanity-plugin-webriq-components",
```

Comment:
Kindly update this with the npm published version of `sanity-plugin-webriq-components` once [packages PR#100](https://github.com/webriq-pagebuilder/sanity-packages/pull/100) is resolved. Thanks!

Reply:
Updated. Thanks!